src/components/tables/teams.js

```diff
@@ -0,0 +1,68 @@
+import T from 'prop-types'
+import Table from './table'
+import Router from 'next/router'
+import join from 'url-join'
+import { useFetchList } from '../../hooks/use-fetch-list'
+import { useState } from 'react'
+import Pagination from '../pagination'
+
+function TeamsTable({ type, orgId }) {
+  const [page, setPage] = useState(1)
+
+  let apiBasePath
+  let emptyMessage
+
+  switch (type) {
+    case 'all-teams':
+      apiBasePath = '/teams'
+      break
+    case 'my-teams':
+      apiBasePath = '/my/teams'
+      emptyMessage = 'You are not part of a team yet.'
+      break
+    case 'org-teams':
+      apiBasePath = `/organizations/${orgId}/teams`
+      emptyMessage = 'This organization has no teams.'
+      break
+    default:
+      break
+  }
+
+  const {
+    result: { data, pagination },
+    isLoading,
+  } = useFetchList(`${apiBasePath}?page=${page}`)
+
+  const columns = [{ key: 'name' }, { key: 'members' }]
+
+  return (
+    <>
+      <Table
+        data-cy={`${type}-table`}
+        rows={data}
+        columns={columns}
+        emptyPlaceHolder={isLoading ? 'Loading...' : emptyMessage}
+        onRowClick={(row) => {
+          Router.push(
+            join(URL, `/team?id=${row.id}`),
```

Comment:
this needs `const URL = process.env.APP_URL` at the top of the file

Reply:
Good catch, fixed.